schema.ts

```diff
@@ -31,13 +31,14 @@ export enum ValueHintType {
   Person = 'person',
   Percent = 'percent',
   Currency = 'currency',
-  Image = 'image',
+  // TODO(spencer): change value to reflect new enum name and update ImageAttachment to Image
```

Comment:
I think it's actually unlikely we'll change the value, we'd have to go rebuild all the packs that use it and bump all their versions, and make the `coda` code handle both values during the transition, so may not be worth the pain.

Reply:
ah okay i can remove this then